### PR TITLE
Move projections into ngeo module

### LINKED ISSUE #789: Move `gmf.proj.EPSG21781` into ngeo module
* Move `gmf.proj.EPSG21781` into ngeo module
 * Remove manual `proj4.defs('EPSG:21781', ...` definitions in examples

### PR DESCRIPTION
**Pull request by tsauerwein**
Closes #789

### CODE REVIEW

File: karma-conf.js [hunk 1]
```diff
@@ -36,7 +36,7 @@ module.exports = function(config) {
       'contribs/gmf/test/spec/beforeeach.js',
       'contribs/gmf/test/spec/data/*.js',
       'contribs/gmf/test/spec/**/*.spec.js',
-      'contribs/gmf/src/proj/epsg21781projection.js',
+      'src/proj/epsg21781projection.js',
```
Comment (gberaudo) [inline karma-conf.js]: Did you add/remove the files into git?

Comment (tsauerwein) [inline karma-conf.js]: Yes, see below.

**Comment by sbrunner** [comment]
Not directly related but should the `epsg2056projection.js` follows whats done for the `21781`?
* low and high case for the 'EPSG'
* better formating of the definition...

File: karma-conf.js (hunk 1 above)
Comment (gberaudo) [inline karma-conf.js]: Good. Didn't see the renaming. +1

**Comment by fredj** [comment]
> low and high case for the 'EPSG'

I'm wondering why we need upper and lower case definitions?

**Comment by fredj** [comment]
@tsauerwein looks good, please merge

**Comment by sbrunner** [comment]
@fredj I don't know, but booth should be the same ...

**Comment by fredj** [comment]
Probably only for https://github.com/camptocamp/ngeo/blob/master/contribs/gmf/examples/mobilebackgroundlayerselector.js#L44

**Pull request merged**